crud.py

```diff
@@ -0,0 +1,69 @@
+from sqlalchemy.orm import Session, joinedload
+
+import models
+import schemas
+
+
+def create_author(
+        db: Session,
+        author: schemas.AuthorCreate,
+) -> models.Author:
+    db_author = models.Author(name=author.name, bio=author.bio)
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+
+    return db_author
+
+
+def get_author_by_name(db: Session, name: str) -> models.Author | None:
+    return db.query(models.Author).filter(models.Author.name == name).first()
+
+
+def get_author(db: Session, author_id: int) -> models.Author | None:
+    queryset = db.query(models.Author)
+
+    return queryset.filter(models.Author.id == author_id).first()
+
+
+def get_authors(
+        db: Session,
+        skip: int = 0,
+        limit: int = 33,
```

Comment:
Why limit is 33?)

Reply:
As for me better to move default limit to separate constant and reuse it in all places

Reply:
All your remarks noted with thanks and the code is fixed accordingly.